drivers/SmartThings/matter-lock/profiles/base-lock-batteryLevel.yml

```diff
@@ -0,0 +1,25 @@
+name: base-lock-batteryLevel
+components:
+- id: main
+  capabilities:
+  - id: lock
+    version: 1
+    config:
+      values:
+      - key: "lock.value"
+        enabledValues:
+        - locked
+        - unlocked
+        - unknown
```

Comment:
Change "unknown" to "not fully locked" in all these new profiles. See https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1602 for reference as to why. With that and the previous related PR the driver will no longer create "unknown" events.

Reply:
fixed.